Jenkinsfile

```diff
@@ -1142,6 +1142,12 @@ pipeline {
                                                sudo mount -t tmpfs -o size=16G tmpfs /mnt/daos
                                                sudo mkdir -p $DAOS_BASE
                                                sudo mount -t nfs $HOSTNAME:$PWD $DAOS_BASE
+
+                                               # copy daos_admin binary into $PATH and fix perms
+                                               sudo cp $DAOS_BASE/install/bin/daos_admin /usr/bin/daos_admin && \
```

Comment:
We could add the bits to scons to do all of that, but it's the wrong tool for the job, IMO. Production installs of DAOS should be done from RPM, and this PR includes specfile updates to handle all of that (including adding a new group).

Reply:
There are two practical problems with that approach:
  1) If I were running a build and it prompted me for a root password, I'd instantly be VERY wary
  2) We can't run sudo on our CI build nodes (and this isn't likely to be a restriction that will be relaxed, due to the possibility of running PRs from external sources, etc.)

Reply:
Does it actually need to be in /usr/bin? I guess that's the question moreso than the assumption that root chowns it and sets appropriate perms.

Reply:
I spent two days trying to resolve the problem of running the helper out of the source tree. In our CI environment, there are a number of irreconcilable issues:

1. The build nodes cannot run sudo, so it is not possible to change the binary's ownership to root and mark it setuid as part of the build process.
1. The test nodes can run sudo, but the build tree is exported via NFS with the root_squash option enabled, which means that root on the test nodes does not have permission to change file ownership or permissions on the in-tree binary.
1. The various test harnesses have a variety of ways of setting the $PATH for their respective test runs. In order to ensure that the correct path to the helper was resolved, I tried prefixing /usr/bin to all of these different environments. I was moderately successful but my whack-a-mole approach did not in the end result in complete success. I wasn't really happy with that approach to begin with, as forcing /usr/bin to be first in the $PATH could have unintended consequences for other tests at some point.

In the end, I decided that the pragmatic approach to resolving this for now was to make some assumptions:

1. Production installs will use RPM (we're not there yet but getting closer all the time)
1. The forwarding failure message should be clear enough to allow the admin who has installed from source to find a resolution in the documentation (which I need to update)
1. Developers have the option of disabling forwarding to preserve existing behavior for now -- this may change at some point if we change the design to make forwarding mandatory (not 100% sure which way this will go, but trying to do things in an incremental fashion)

Reply:
I've thought about it some more, and you're right to push back on this. It's not even necessarily correct for RPM-based installs. It's entirely possible that someone might use the --prefix or --relocate options when installing, in which case the static path will be wrong. Occurred to me that I might be able to just move the in-tree binary out of the way (or remove the execute bits I guess) so that the /usr/bin version is found in CI without needing to mess with $PATH. Fingers crossed...